Formula/svgbob.rb

```diff
@@ -0,0 +1,34 @@
+class Svgbob < Formula
+  desc "Convert your ascii diagram scribbles into happy little SVG"
+  homepage "https://ivanceras.github.io/svgbob-editor/"
+  url "https://github.com/ivanceras/svgbob/archive/0.5.0.tar.gz"
+  sha256 "731de78542ad5bbe7400711e1b3b98ef191506b766bd36745629108356d312b1"
+  license "Apache-2.0"
+  head "https://github.com/ivanceras/svgbob.git"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
```

Comment:
```suggestion
```

Reply:
no need to have `livecheck` yet

Reply:
After reading @samford's thoughts on the given topic https://github.com/Homebrew/homebrew-core/pull/80167#issuecomment-870232507 I started adding livechecks everywhere because it seems like the right thing to do?

Reply:
We only need to add a `livecheck` block if the default behaviour (what you get from `brew livecheck` with no `livecheck` block) is incorrect.

Reply:
It's true that we only need to add a `livecheck` block when livecheck's default behavior doesn't produce a good result but there's maybe something to be said for taking a more proactive approach with the `Git` strategy. Up to this point, we've generally only added a `livecheck` block to a formula using the `Git` strategy when something breaks (e.g., a weird tag appears that causes a problem).

Relying on default behavior is preferable for strategies that generate a more specific regex using information from the URL (e.g., `Apache`, `Bitbucket`, `Cpan`, etc.). However, the `Git` strategy's default regex simply omits non-digit text from the start of a tag, so it's very permissive. As explained in the linked comment, there's quite a bit of variety with tag formats, so it's desirable for the `Git` strategy's default regex to be very permissive (i.e., using the standard regex for Git tags like `1.2.3`/`v1.2.3` as default wouldn't be a good idea overall).

That said, I'm of two minds about proactively adding a regex to formalize the tag format before we encounter an issue with the default `Git` regex. On the one hand, if we have a solid feel for the tag/version format, adding a regex could potentially avoid unwanted tags in the future without waiting for it to break. On the other hand, if there aren't many tags to reference when trying to understand the tag format, we may end up using a regex that will omit tags in the future that we should be matching (e.g., we see tags like `1.0`, `1.1` but the version format could allow for `1.1b`, `1.1c`, etc.).

Despite my previous comment, I would prefer to continue omitting unnecessary `Git` `livecheck` blocks when it's not technically necessary and there aren't enough tags to definitively establish the tag/version format. However, that's kind of subjective, so I think it's easiest if we continue to only add `Git` `livecheck` blocks when we encounter an issue, at least for now.

It may be that by trying to proactively avoid issues, we could end up creating a different set of issues. The default `Git` behavior generally works fine in practice and we don't run into too many issues, so it's less work overall to just address breakages as they occur.

@vladimyr I appreciate the effort with the `livecheck` block but let's hold off on this idea for now. We may revisit this in the future if/when we create a system to create automated version bumps but it may be more trouble than it's worth until then.